test/test_scanner.py

```diff
@@ -1139,10 +1139,10 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "1.8.0",
                     ),
                     (
-                        "https://kojipkgs.fedoraproject.org/packages/postgresql/9.4.4/1.fc22/x86_64/",
-                        "postgresql-9.4.4-1.fc22.x86_64.rpm",
-                        "postgresql",
-                        "9.4.4",
```

Comment:
Why was `postgresql` removed?

Reply:
I guess it was removed unintentionally when I was removing v9.7.1